docs/affiliates/about/index.html

```diff
@@ -95,7 +95,7 @@
 
 
         <li class="relative">
-          <a href="../../who-we-are" id="who-we-are-nav-item">Who We Are</a>
+          <a href="../../who-we-are" id="who-we-are-nav-item">About</a>
```

Comment:
do you want to rename the site directory for `who-we-are` to `about` to make this consistent?

Reply:
I left it to keep links alive, we may want to alias at some point.